### DISCUSSION TITLE
My account got suspended without any reason and no response from GitHub contact

### DISCUSSION THREAD

**Discussion by bunty34** [community/community#134137] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

In Sunday morning my account got suspended while commenting on a discussion page 📃 
Still no response from the GitHub support team please help me. 
My profile is - @RamakrushnaBiswal. Please help me to get my account back account 🙏 
I have all of my projects and all things are there...

Thank you @github

**Comment by bunty34** [comment]
hi @queenofcorgis @ghostinhershell @davidstaheli any one help me to recover my account

my account link https://github.com/RamakrushnaBiswal

**Reply by bunty34** [reply]
2904697  
tocket id

**Comment by CN-lyb** [comment]
try to make a new one,if you do not have impotant things.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @bunty34, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

_Please note: contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our Code of Conduct for more information._

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time._

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
